app/src/main/kotlin/com/wire/android/ui/WireActivity.kt

```diff
@@ -195,6 +225,18 @@ class WireActivity : AppCompatActivity() {
         super.onPause()
         proximitySensorManager.unRegisterListener()
     }
+
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        // FIXME do not use deprecated onActivityResult
+        //  when the issue https://issuetracker.google.com/issues/181785653 will be fixed
+        if (requestCode == WireAppUpdateManager.REQUEST_CODE) {
+            if (resultCode != RESULT_OK) {
```

Comment:
So if the `resultCode` is `RESULT_OK` then it means that the update wasn't completed successfully? Or should it be reversed? 😄

Reply:
damn, forgot to remove `!` ;P